sentry-ruby/lib/sentry/log_event.rb

```diff
@@ -109,5 +126,34 @@ def value_type(value)
         "string"
       end
     end
+
+    def parameters
+      @parameters ||= begin
+        return DEFAULT_PARAMETERS unless template
+
+        parameters = template_tokens.empty? ?
+          attributes.fetch(:parameters, DEFAULT_PARAMETERS) : attributes.slice(*template_tokens)
+
+        if parameters.is_a?(Hash)
+          parameters.each do |key, value|
+            attributes["sentry.message.parameters.#{key}"] = value
+          end
+        else
+          parameters.each_with_index do |param, index|
+            attributes["sentry.message.parameters.#{index}"] = param
```

Comment:
they just changed this to `sentry.message.parameter'
see https://github.com/getsentry/sentry-python/pull/4387

Reply:
something something hardest problem in computer science